### PR TITLE
[Merged by Bors] - Support Trino v414

### PR DESCRIPTION
**Pull request by maltesander**
# Description

test: https://ci.stackable.tech/view/02%20Operator%20Tests%20(custom)/job/trino-operator-it-custom/68/

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes
 
```[tasklist]
# Author
- [ ] Changes are OpenShift compatible
- [ ] CRD changes approved
- [ ] Helm chart can be installed and deployed operator works
- [ ] Integration tests passed (for non trivial changes)
```

```[tasklist]
# Reviewer
- [ ] Code contains useful comments
- [ ] (Integration-)Test cases added
- [ ] Documentation added or updated
- [ ] Changelog updated
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
```

```[tasklist]
# Acceptance
- [ ] Feature Tracker has been updated
- [ ] Proper release label has been added
```

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +16,7 @@ All notable changes to this project will be documented in this file.
 
 [#419]: https://github.com/stackabletech/trino-operator/pull/419
 [#421]: https://github.com/stackabletech/trino-operator/pull/421
+[#422]: https://github.com/stackabletech/trino-operator/pull/422
```
Comment (sbernauer) [inline CHANGELOG.md:19]:
```suggestion
[#423]: https://github.com/stackabletech/trino-operator/pull/423
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Added
+
+- Support for Trino v414 ([#422]).
```
Comment (sbernauer) [inline CHANGELOG.md:9]:
```suggestion
- Support for Trino v414 ([#423]).
```

**Review by sbernauer — APPROVED** [review]
Thx!

**Comment by maltesander** [comment]
bors merge

**Pull request closed without merging**